server/api/swagger/admin.yaml

```diff
@@ -4711,5 +4711,167 @@ paths:
       x-security-types:
         - bearer
         - hmac
+      x-security-scopes:
+        - admin
+  /admin/roles:
```

Comment:
I don't see you changing any of the existing swagger securities from the hardcoded ones we already got